ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -1339,6 +1339,42 @@ def test_tensorflow_crelu(
     )
 
 
+# conv_transpose
+@handle_frontend_test(
+    fn_tree="tensorflow.nn.conv_transpose",
+        x_f_d_df=_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["NWC", "NHWC"]),
+        padding=st.sampled_from(["SAME", "VALID"]),
+        dilation_max=1,
+        type=None,
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_conv_transpose(
+    *,
+    x_f_d_df,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x, filters, dilation, data_format, stride, padding = x_f_d_df
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x,
```

Comment:
There is no such thing as `input` in your function. rename this to `value`

Reply:
Hey @Aarsh2001, changed `input` to `value`.